web/static/js/group-management.js

```diff
@@ -0,0 +1,169 @@
+/* Set up the necessary arrays and populate them */
+var streams = [];
+var zones = [];
+var groups = [];
+updateSettings();
```

Comment:
I think all of this should be merged into one `settings.js` file since there is a lot of overlap